app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/view_return.scala.html

```diff
@@ -160,7 +160,14 @@ <h2 class="govuk-heading-l">
         @messages(s"$key.summaryHeading")
     </h2>
 
-    @changeReturnLink("amend-link-1")
+    @if(returnSummary.expired) {
+        <p class="govuk-inset-text">
+            @messages("viewReturn.expired")
+        </p>
+    } else {
+        @changeReturnLink("amend-link-1")
+    }
+
```

Comment:
there is another changeReturnLink in this template that will continue to show https://github.com/hmrc/cgt-property-disposals-frontend/blob/eb31d93672bd7b4e3ee9db6a255f17edcf581c89/app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/view_return.scala.html#L183